ios/FluentUI/Other Cells/CenteredLabelCell.swift

```diff
@@ -8,23 +8,62 @@ import UIKit
 // MARK: CenteredLabelCell
 
 @objc(MSFCenteredLabelCell)
-open class CenteredLabelCell: UITableViewCell {
+open class CenteredLabelCell: UITableViewCell, TokenizedControlInternal {
```

Comment:
A lot of these controls are UITableViewCells, not our TableViewCell. Should they actually be using the same tokens as the TableViewCell? Or should they be updated to inherit from our TableViewCell rather than the UIKit UITableViewCell?